CONTRIBUTING.md

```diff
@@ -1,425 +1,365 @@
-
-  ```sh
-  $ git push origin my-feature-branch
-  ```
-
-
-#### Agree to the contributor License
-
-Before we can accept and merge your changes, you must sign a [Contributor License Agreement](http://developer.marklogic.com/products/cla). You only need to do this once.
-
-
-#### Submit the pull request
-
-Go to https://github.com/username/marklogic-data-hub and select your feature branch. Click
-the 'Pull Request' button and fill out the form.
-
-Pull requests are usually reviewed within a few days. If you get comments
-that need to be to addressed, apply your changes in a separate commit and push that to your
-feature branch. Post a comment in the pull request afterwards; GitHub does
-not send out notifications when you add commits to existing pull requests.
-
-That's it! Thank you for your contribution!
-
-
-#### After your pull request is merged
-
-After your pull request is merged, you can safely delete your branch and pull the changes
-from the main (upstream) repository:
-
-* Delete the remote branch on GitHub either through the GitHub web UI or your local shell as follows:
-
-    ```shell
-    git push origin --delete my-feature-branch
-    ```
-
-* Check out the develop branch:
-
-    ```shell
-    git checkout develop -f
-    ```
-
-* Delete the local branch:
-
-    ```shell
-    git branch -D my-feature-branch
-    ```
-
-* Update your develop with the latest upstream version:
-
-    ```shell
-    git pull --ff upstream develop
-    ```
-
-[issue tracker]: https://github.com/marklogic/marklogic-data-hub/issues
-[.editorconfig]: http://editorconfig.org/
+# Contributing to MarkLogic Data Hub
+
+MarkLogic Data Hub welcomes new contributors. This document will guide you
+through the process.
+
+ - [Issues and Bugs](#found-an-issue)
+ - [Feature Requests](#want-a-feature)
+ - [Developing Data Hub](#developing-data-hub)
+ - [Submission Guidelines](#submission-guidelines)
+
+
+## Found an Issue?
+If you find a bug in the source code or a mistake in the documentation, you can help us by submitting an issue to our [GitHub Issue Tracker][issue tracker]. Even better, you can submit a Pull Request with a fix for the issue you filed.
+
+
+## Want a Feature?
+You can request a new feature by submitting an issue to our [GitHub Issue Tracker][issue tracker].  If you
+would like to implement a new feature then first create a new issue and discuss it with one of our
+project maintainers.
+
+
+## Developing Data Hub 
+
+This section describes how to build and test new features and fixes in Data Hub. This includes changes to the following subprojects in this project:
+
+- the Data Hub library in ./marklogic-data-hub
+- the Data Hub Gradle plugin in ./ml-data-hub-plugin
+- the One-UI web application in ./one-ui
+- the QuickStart web application in ./web
+
+### Prerequisites
+
+You need:
+
+- MarkLogic Server (see [Version Compatibility](https://docs.marklogic.com/datahub/refs/version-compatibility.html) for the correct version)
+- Java JDK 8 or later
```

Comment:
Should It be Java 8 or 11?

Reply:
Good catch - technically, it should be Java 9, as that's what is in our build.gradle files. I'll update.

Reply:
@rjrudin Can we build the project using java9? currently we are building using 11.

Reply:
We can - our build.gradle files have source and target compatibilities of Java 9.

Reply:
In practice though, people are likely using Java 11.